esignet-ida115-integration-impl/src/test/java/io/mosip/ida115/esignet/integration/service/IdaAuthenticatorImplTest.java

```diff
@@ -62,7 +64,12 @@ public class IdaAuthenticatorImplTest {
 
 	@Mock
 	ObjectMapper mapper;
-
+	
+	ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
if creating a new instance then lets delete the mock instance created in line 66

Reply:
Applied the change as per the review comment.